src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp

```diff
@@ -506,6 +506,55 @@ PYBIND11_MODULE(simplnx, mod)
       },
       "Convert numpy dtype to simplnx DataType", "dtype"_a);
 
+  mod.def(
+      "convert_np_dtype_to_numerictype",
+      [](const py::dtype& dtype) {
+        if((dtype.attr("__eq__")(py::dtype::of<int8>())).cast<bool>())
```

Comment:
I think since we're using this in multiple places now it makes sense to make a helper function.
```cpp
/**
 * @brief Equivalent to lhs.__eq__(rhs) in python
 * @param lhs
 * @param rhs
 * @return bool
 */
bool PyIsEqual(py::handle lhs, py::handle rhs)
{
  return (lhs.attr("__eq__")(rhs)).cast<bool>();
}
```